prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -260,10 +257,9 @@ class ReportFunction : Logging {
      * @param request the http request message from the client
      */
     private fun extractClientHeader(request: HttpRequestMessage<String?>): String {
```

Comment:
I didn't see that the value returned by extractClientHeader was ever being used by any Okta related stuff, so I removed the ".replace" stuff here.